pkg/asset/installconfig/aws/permissions.go

```diff
@@ -5,192 +5,224 @@ import (
+
+		// ELB related perms
+		"elasticloadbalancing:AddTags",
+		"elasticloadbalancing:ApplySecurityGroupsToLoadBalancer",
+		"elasticloadbalancing:AttachLoadBalancerToSubnets",
+		"elasticloadbalancing:CreateListener",
+		"elasticloadbalancing:CreateLoadBalancer",
+		"elasticloadbalancing:CreateLoadBalancerListeners",
+		"elasticloadbalancing:CreateTargetGroup",
+		"elasticloadbalancing:ConfigureHealthCheck",
+		"elasticloadbalancing:DeleteLoadBalancer",
+		"elasticloadbalancing:DeregisterInstancesFromLoadBalancer",
+		"elasticloadbalancing:DeregisterTargets",
+		"elasticloadbalancing:DescribeInstanceHealth",
+		"elasticloadbalancing:DescribeListeners",
+		"elasticloadbalancing:DescribeLoadBalancers",
+		"elasticloadbalancing:DescribeLoadBalancerAttributes",
+		"elasticloadbalancing:DescribeTags",
+		"elasticloadbalancing:DescribeTargetGroupAttributes",
+		"elasticloadbalancing:DescribeTargetHealth",
+		"elasticloadbalancing:ModifyLoadBalancerAttributes",
+		"elasticloadbalancing:ModifyTargetGroup",
+		"elasticloadbalancing:ModifyTargetGroupAttributes",
+		"elasticloadbalancing:RegisterTargets",
+		"elasticloadbalancing:RegisterInstancesWithLoadBalancer",
+		"elasticloadbalancing:SetLoadBalancerPoliciesOfListener",
+
+		// IAM related perms
+		"iam:AddRoleToInstanceProfile",
+		"iam:CreateInstanceProfile",
+		"iam:CreateRole",
+		"iam:DeleteInstanceProfile",
+		"iam:DeleteRole",
+		"iam:DeleteRolePolicy",
+		"iam:GetInstanceProfile",
+		"iam:GetRole",
+		"iam:GetRolePolicy",
+		"iam:GetUser",
+		"iam:ListInstanceProfilesForRole",
+		"iam:ListRoles",
+		"iam:ListUsers",
+		"iam:PassRole",
+		"iam:PutRolePolicy",
+		"iam:RemoveRoleFromInstanceProfile",
+		"iam:SimulatePrincipalPolicy",
+		"iam:TagRole",
+
+		// Route53 related perms
+		"route53:ChangeResourceRecordSets",
+		"route53:ChangeTagsForResource",
+		"route53:GetChange",
+		"route53:GetHostedZone",
+		"route53:CreateHostedZone",
+		"route53:DeleteHostedZone",
+		"route53:ListHostedZones",
+		"route53:ListHostedZonesByName",
+		"route53:ListResourceRecordSets",
+		"route53:ListTagsForResource",
+		"route53:UpdateHostedZoneComment",
+
+		// S3 related perms
+		"s3:CreateBucket",
+		"s3:DeleteBucket",
+		"s3:GetAccelerateConfiguration",
+		"s3:GetBucketCors",
+		"s3:GetBucketLocation",
+		"s3:GetBucketLogging",
+		"s3:GetBucketObjectLockConfiguration",
+		"s3:GetBucketReplication",
+		"s3:GetBucketRequestPayment",
+		"s3:GetBucketTagging",
+		"s3:GetBucketVersioning",
+		"s3:GetBucketWebsite",
+		"s3:GetEncryptionConfiguration",
+		"s3:GetLifecycleConfiguration",
+		"s3:GetReplicationConfiguration",
+		"s3:ListBucket",
+		"s3:PutBucketAcl",
+		"s3:PutBucketTagging",
+		"s3:PutEncryptionConfiguration",
+
+		// More S3 (would be nice to limit 'Resource' to just the bucket we actually interact with...)
+		"s3:PutObject",
+		"s3:PutObjectAcl",
+		"s3:PutObjectTagging",
+		"s3:GetObject",
+		"s3:GetObjectAcl",
+		"s3:GetObjectTagging",
+		"s3:GetObjectVersion",
+		"s3:DeleteObject",
```

Comment:
this would go under both `create-base` (because we sometimes remove after bootstrap-complete) and `delete-base` (because we sometimes remove during `destroy cluster`)